WalletWasabi.Tests/UnitTests/Wallet/SmartCoinSelectorTests.cs

```diff
@@ -163,12 +163,13 @@ private List<SmartCoin> GenerateSmartCoins(IEnumerable<(string Cluster, decimal
 		{
 			var key = KeyManager.GenerateNewKey(new LabelsArray(targetCoin.Cluster), KeyState.Clean, false);
 
-			if (!generatedKeyGroup.ContainsKey(targetCoin.Cluster))
+			if (!generatedKeyGroup.TryGetValue(targetCoin.Cluster, out List<(HdPubKey key, decimal amount)>? value))
```

Comment:
Rider always recommends here to use `var`. I know @kiminuo prefers very much to be specific all the time. Personal choice I guess, it's a discussion we're having all the time. Feel free to leave it as you did, simply a comment.